src/EnergyPlus/SolarShading.cc

```diff
@@ -485,6 +550,17 @@ void GetShadowingInput(EnergyPlusData &state)
         state.dataSysVars->shadingMethod = ShadingMethod::PolygonClipping;
     }
 
+    if (state.dataSysVars->shadingMethod == DataSystemVariables::ShadingMethod::PixelCounting &&
+        state.dataSolarShading->anyScheduledShadingSurface == true) {
+        ShowSevereError(state, "The Shading Calculation Method of choice is \"PixelCounting\";");
+        ShowContinueError(state, "and there is at least one shading surface of type ");
+        ShowContinueError(state, "Shading:Site:Detailed, Shading:Building:Detailed, or Shading:Zone:Detailed,");
+        ShowContinueError(state, "that has an active transmittance schedule value greater than zero.");
+        ShowContinueError(state, "With \"PixelCounting\" Shading Calculation Method, the shading surfaces will be treated as completely opaque (transmittance = 0) during the shading calculation, ");
+        ShowContinueError(state, "which may result in inaccurate or unexpected results.");
+        ShowContinueError(state, "It is suggested to use another Shading Calculation Method, such as \"PolygonClipping\".");
+    }
+
     aNum++;
```

Comment:
Implemented with the method suggested by the reviewer. The severe warning will now be reported only once; and only for shading surfaces that ACTUALLY has some transmittance value great than 0.0.

Reply:
The method to report this method is as clean as could be expected. This has been tested and works as expected.